### PR TITLE
feature: add feeds script [dev]

### PR DESCRIPTION
**Pull request by jjordan-quantum**
Created npm project and scripts for adding equity symbols to feeds.yaml (currently adds to feeds.yaml in /dev/bsc/ folder)

### CODE REVIEW

File: test/test.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  console.log('Length of feeds.yaml after restoring from backup: ' + feedsYamlDataLength_2 + ' keys');
+  console.log('Length of feeds-55.yaml after restoring from backup: ' + feedsYaml55DataLength_2 + ' keys');
+})()
+  .then(() => {
+    process.exit(0);
+  })
+  .catch((error) => {
+    console.error(error);
+    process.exit(1);
+  });
```
Comment (dougmrqs) [inline test/test.ts:50]: We usually use a framework like `chai` to assure everything works, but this use case would make a good example. Not sure about the time we want to spend on this task, but I think the goal is achieved. If we had more time, I'd recommend we implement a testing framework here.

Other than calling it a "test", we could call this an "example".

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "main": "src/main.js",
+  "scripts": {
+    "lint": "eslint ./src/**/*.ts ./test/**/*.ts --max-warnings=6",
+    "lint:fix": "eslint ./src/**/*.ts ./test/**/*.ts --fix",
+    "prettier": "prettier --config .prettierrc ./src/**/*.ts ./test/**/*.ts --check",
+    "prettier:format": "prettier --config .prettierrc ./src/**/*.ts ./test/**/*.ts --write",
+    "format": "npm run lint:fix && npm run prettier:format",
+    "start": "tsc src/main.ts && cd src && node main.js",
+    "restore": "tsc src/restoreFeeds.ts && cd src && node restoreFeeds.js",
+    "test": "tsc test/test.ts && cd test && node test.js"
```
Comment (dougmrqs) [inline package.json:14]: Please mention this in the readme. Consider calling this script an example instead of a proper test.

**Review by dougmrqs — CHANGES_REQUESTED** [review]

**Review by dougmrqs — APPROVED** [review]
Looks good enough to me. Just some final hints to keep in mind: 
- We branch from the repository itself, not from a fork;
- We usually name our branches in the git/hubflow pattern, like "feature", "hotfix" and "release".
- We'll have to copy this script to the main branch, given our current strategy: develop branch serves feeds for develop and sandbox environment and main branch serves feeds for production environment.
- For this case, I'd name this branch a `feature/add-feeds-script`
- Should we update the feeds.yaml here in this PR and upload the updated feeds file?

**Comment by jjordan-quantum** [comment]
Thanks Douglas,
I will branch from the main repo for prod where I need to duplicate this code.
I will update the feeds here and upload them with another commit to the upgraded PR.
Once the code is duplicated in the new branch for prod, I will update the yaml there as well

**Pull request merged**